Quadriga/src/main/java/edu/asu/spring/quadriga/service/projectblog/IProjectBlogEntryManager.java

```diff
@@ -0,0 +1,41 @@
+package edu.asu.spring.quadriga.service.projectblog;
+
+import java.util.List;
+
+import edu.asu.spring.quadriga.domain.projectblog.IProjectBlogEntry;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.impl.projectblog.ProjectBlogEntryManager;
+
+/**
+ * This interface provides template for project blog entry manager
+ * implementations.
+ * 
+ * @author PawanMahalle
+ *
+ * @see ProjectBlogEntryManager
+ */
+public interface IProjectBlogEntryManager {
+
+    /**
+     * populates the details of a project blog entry and uses DAO to add an
```

Comment:
You're missing the important things here. The method generates a new ID for the blog entry and it sets its date. Also, remove the table stuff. This is a manager class that doesn't know anything about where things are actually stored.